pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -51,6 +51,8 @@ Secrets should include paths, unless you want to allow all services to access it
 
 1. Click **Create**.
 
+**Note:** It is not possible to create a binary secret using the GUI. Only the CLI can be used to create binary secrets.
```

Comment:
Is the user expected to know what a binary secret is at this point? Maybe rearrange the doc or link to a definition?